include/deal.II/lac/la_parallel_vector.templates.h

```diff
@@ -21,6 +21,7 @@
 
 #include <deal.II/base/cuda.h>
 #include <deal.II/base/cuda_size.h>
+#include <deal.II/base/partitioner.templates.h>
```

Comment:
?

Reply:
Compilation fails if I use a data type for `LA:d:V` that it was not specialized for.

Reply:
As this is unrelated to this PR if we use `float`, I'll revert this change.